### ISSUE TITLE
Building in Windows 7 - CA Certificate Installation Issue

### ISSUE THREAD

**Issue by smozgur** [OfficeDev/Excel-Custom-Functions#168]
Building works without problems in Windows 10, however there is a certificate installation issue in Windows 7 Home Premium.

"The term Import-Certificate is not recognized as the name of a cmdlet..." 
<img width="959" alt="image" src="https://user-images.githubusercontent.com/7944159/60110215-694a7a80-9774-11e9-9125-d034ebaf9779.png">

I do understand Import-Certificate is not a Windows 7 powershell cmdlet.

I tried to install the generated CA certificate manually, however build script still regenerates it and tries to install with powershell, and cannot pass that step that avoids testing Excel Custom Functions in Windows 7.

<img width="1048" alt="image" src="https://user-images.githubusercontent.com/7944159/60112217-14106800-9778-11e9-9ab7-edfbf5a57020.png">

I disabled certificate installation in node_modules\office-addin-dev-certs\lib\install.js, and yes it skipped the installation, but add-in loading failed, likely due to missing certification.

Can't we use Windows 7 for testing Excel Custom Functions with Node.js (other than Trusted Add-in Catalog), or is there a workaround that I am probably missing?

**Comment by asachin96** [comment]
Hi, 

Can you please let me know what is your PowerShell version. You can use the following command
$PSVersionTable.PSVersion.Major

**Comment by smozgur** [comment]
@asachin96, thanks for your reply. 

Windows 7 - PS version is 2.
If we somehow update PS to 4, then cmdlet will be also available?

**Comment by asachin96** [comment]
Import-Certificate is available from PowerShell 3. Yes, upgrading PowerShell should solve this issue.
Here's a doc to upgrade PS https://docs.microsoft.com/en-us/powershell/scripting/install/installing-windows-powershell?view=powershell-6. 
Let me know how it goes. I'm going to add PowerShell version 3 requirement to certificate package.

**Comment by smozgur** [comment]
I actually didn't try to upgrade when I saw this earlier, and it made sense:
https://superuser.com/questions/1353868/powershell-is-missing-import-certificate

But I will try now. I'll let you know. Thank you.

**Comment by smozgur** [comment]
I tried version 3, then 4. No luck.
Used Get-Module -ListAvailable .... to list available modules. There is no PKI with Import-Certificate in Windows 7 which actually shows up in Windows 10.

"Module tied to the OS version instead WMF" in the link above makes more sense now.

**Comment by asachin96** [comment]
I see. I'll check and get back to you.

**Comment by smozgur** [comment]
Ok, I have solved this by installing the CA certificate (%userprofile%/.office-addin-dev-certs/ca.crt) into the Trusted Root Certificate Authorities store. 
It didn't work initially as I said in my initial question above, because I let it to install the certificate into the default cert store. I searched for the cert and it was not located in this store, so I installed it again by specifying the cert store.

I don't think PowerShell will help for Windows 7, unless the install.js script is perhaps updated to use certutil in Windows 7. Not sure, but I wanted to send this workaround for users trying to use Excel Custom Functions in Windows 7.
